build/AzurePipelinesTemplates/WindowsAppSDK-PackTransportPackage-Stage.yml

```diff
@@ -142,6 +142,28 @@ stages:
       env:
         LIB: $(Build.SourcesDirectory)
 
+    # In the "Publish symbols to SymWeb and MSDL" step, the symbols are automatically stripped in the API.
+    # However, since WindowsAppSDK is public, we don't need to strip them.
+    # So we need to send a list of pdbs to the symbols team that is not needed to be stripped
+    # This task facilitates this.
+    - task: PowerShell@2
+      name: ListUniquePDBs
+      displayName: 'List Unique PDBs and write to pdbs.txt'
+      inputs:
+        targetType: 'inline'
+        script: |
+          $pdbSet = @{}
+          $inputPath = "$(Build.ArtifactStagingDirectory)\symbols"
+          Get-ChildItem *.pdb -Path $inputPath -Recurse | foreach-object {
+            $pdbSet[$_.Name] = $true
+          }
+          foreach ($value in $pdbSet.Keys)
+          {
+            Write-Host $value
+          }
+          New-Item "$(ob_outputDirectory)\pdbs.txt" -Force
+          $pdbSet.Keys | Out-File "$(ob_outputDirectory)\pdbs.txt"
```

Comment:
You can replace lines 164-165 with the simpler

```
$pdbSet.Keys | Set-Content -Path "$(ob_outputDirectory)\pdbs.txt" -Force
```

You can also control encoding if need be (see other Set-Content uses across the repo for examples)